presto-docs/src/main/sphinx/release/release-0.185.rst

```diff
@@ -8,14 +8,20 @@ General Changes
 * Fix incorrect column names in ``QueryCompletedEvent``.
 * Fix excessive CPU usage in coordinator for queries that have
   large string literals containing non-ASCII characters.
+* Fix potential infinite loop during query optimization when constant
+  expressions fail during evaluation.
+* Fix incorrect ordering when the same field appears multiple times
+  with different ordering specifications in a window function ``ORDER BY``
+  clause. E.g., ``OVER (ORDER BY x ASC, x DESC)``.
```

Comment:
Let's write it "For example: ..." to match the other one (having an entire sentence consist of "e.g." looks a bit weird)

Reply:
Sure, but "e.g" means exactly "for example", and that form usage is perfectly normal.